src/test/ui/regions/regions-nested-fns-2.nll.stderr

```diff
@@ -2,15 +2,16 @@ error[E0597]: `y` does not live long enough
   --> $DIR/regions-nested-fns-2.rs:18:25
    |
 LL |         |z| {
-   |         --- value captured here
+   |          - has type `&'0 isize`
 LL |             //~^ ERROR E0373
 LL |             if false { &y } else { z }
-   |                         ^ borrowed value does not live long enough
+   |                         ^ `y` would have to be valid for `'0`
 LL |         });
 LL | }
-   | - `y` dropped here while still borrowed
+   | - ...but `y` is only valid for the duration of the `nested` function, so it is dropped here while still borrowed
```

Comment:
Hmm, this message doesn't quite seem right, does it? `y` is valid for the function `foobar`, not "the nested function", which sounds like the closure.

Reply:
What function `foobar`? In this test case, the function is called `nested`.

https://github.com/rust-lang/rust/blob/078fa7fad4076c0738206e410a99ca778193cfba/src/test/ui/regions/regions-nested-fns-2.rs#L11-L20